pingouin/multivariate.py

```diff
@@ -326,22 +326,20 @@ def boxM (covs,sizes, alpha=.001):
         
     # calculate C
     k1 = (2*covs.shape[1]^2+3*covs.shape[1]-1)/(6*(covs.shape[1]+1)*(covs.shape[0]-1))
-    k2 = ((covs.shape[0]+1)*(2*covs.shape[1]^2+3*covs.shape[1]-1))/(6*covs.shape[0]*(covs.shape[1]+1)*(np.sum(sizes)/covs.shape[0]-1))
+    k2 = -((covs.shape[0]+1)*(2*covs.shape[1]**2+3*covs.shape[1]-1))/(6*covs.shape[0]*(covs.shape[1]+1)*(np.sum(sizes)/covs.shape[0]-1))
     T=0
     if (sizes==sizes.mean()).all():
-        C=k2
+        c=-k2
     else:
         for idx_cov in range(covs.shape[0]):
-            T+=(1/(sizes[idx_cov]-1))
-        c=k1*(T-(1/(np.sum(sizes)-covs.shape[0])))
-        
+            T=-T+(1/(sizes[idx_cov]-1))
+        c=-k1*(T-(1/(np.sum(sizes)-covs.shape[0])))      
     # calculate U statistics and degree of fredom
     u =-2*(1-c)*np.log(M)
+    
     df = 0.5*covs.shape[1]*(covs.shape[1]+1)*(covs.shape[0]-1)
     p = 1-scipy.stats.chi2.cdf(u,df)
     equal_cov = True if p > alpha else False
     stats = pd.DataFrame(data={'Chi2':[u], 'df':[df],'pval':[p],'equal_cov':[equal_cov]},index=["Box's M"])
```

Comment:
Could you please rename the index to `box` instead of `Box's M`? We want to avoid having "'" in index names.